rotkehlchen/history/events/structures/asset_movement.py

```diff
@@ -31,14 +31,26 @@
     from rotkehlchen.accounting.mixins.event import AccountingEventMixin
     from rotkehlchen.accounting.pot import AccountingPot
     from rotkehlchen.fval import FVal
+    from rotkehlchen.types import Fee
 
 logger = logging.getLogger(__name__)
 log = RotkehlchenLogsAdapter(logger)
 
 
-class AssetMovement(HistoryBaseEntry):
+class AssetMovementExtraData(TypedDict):
+    address: NotRequired[str]
+    transaction_id: NotRequired[str]
+    reference: NotRequired[str]
+    movement_id: NotRequired[str]
```

Comment:
where have movement_id and reference. Should we just use reference? (reference is how I named what was previously link and url by you)

Reply:
Yes, `movement_id` should probably be replaced with `reference`. No need to have more different keys than necessary.